package.json

```diff
@@ -28,21 +28,25 @@
     "rimraf": "^2.6.1",
     "ts-loader": "^2.0.1",
     "typescript": "^2.1.6",
-    "zone.js": ">=0.7.2"
-  },
-  "dependencies": {
-    "@angular/common": ">=2.0.0",
-    "@angular/compiler": ">=2.0.0",
-    "@angular/core": ">=2.0.0",
-    "@angular/platform-browser": ">=2.0.0",
-    "@angular/platform-browser-dynamic": ">=2.0.0",
+    "@angular/common": "4.0.2",
```

Comment:
Whatever goes into `devDependencies` is for development only but you need to make sure you provide proper dependencies for lib clients. Also, since there are no tests, you do not need angular 4 as your devDependency.